addons/mail/models/mail_activity_mixin.py

```diff
@@ -92,6 +92,22 @@ def _default_activity_type(self):
         help="Type of the exception activity on record.")
     activity_exception_icon = fields.Char('Icon', help="Icon to indicate an exception activity.",
         compute='_compute_activity_exception_type')
+    activity_plans_ids = fields.Many2many(
+        'mail.activity.plan',
+        string="Activity Plans",
+        compute='_compute_activity_plans_ids',
+        search='_search_activity_plans_ids',
+    )
+
+    @api.depends('activity_ids.activity_plan_id')
+    def _compute_activity_plans_ids(self):
+        for record in self:
+            record.activity_plans_ids = record.activity_ids.mapped('activity_plan_id')
```

Comment:
Was `related="activity_ids.activity_plan_id"` not ok? (not sure how that behaves with active_test to be honest, maybe not)

Also fyi `record.activity_ids.mapped('activity_plan_id')` == `record.activity_ids.activity_plan_id` (mostly it's just shorter :shrug: )